components/observations/uploader/uploadObservation.ts

```diff
@@ -8,12 +8,17 @@ import {apiDateString} from 'utils/date';
 export async function uploadObservation(id: string, data: ObservationTaskData): Promise<Observation> {
   const {formData, extraData} = data;
   const {url, ...params} = extraData;
+  // The user only picks a date, not a time. The time defaults to whatever the current time is in the user's timezone.
+  // If the time is late enough, it might be in the next day when converted to UTC. By setting the time to noon in the user's
+  // local timezone, we mitigate this problem.
+  const adjustedDate = new Date(formData.start_date);
+  adjustedDate.setHours(12, 0, 0, 0);
   const payload: Partial<Observation> = {
     ...formData,
     ...params,
     obs_source: 'public',
     // Date has to be a plain-old YYYY-MM-DD string
-    start_date: apiDateString(formData.start_date),
+    start_date: apiDateString(adjustedDate),
```

Comment:
Could we simply do the formatting without converting to UTC? Might be simpler and in case we ever get observations from the Middle East, would still work ;)

Reply:
yes, that's cleaner - I like it